ql/src/java/org/apache/hadoop/hive/ql/ddl/database/desc/DescDatabaseFormatter.java

```diff
@@ -104,6 +113,14 @@ void showDatabaseDescription(DataOutputStream out, String database, String comme
           out.write(ownerType.name().getBytes(StandardCharsets.UTF_8));
         }
         out.write(Utilities.tabCode);
+        if (connectorName != null) {
```

Comment:
nit picking : but why <variable> != null here and null != <variable> previously ?  Can you please consider sticking to one style ?

Also java.util.Objects.isNull can used to check for null

Reply:
fixed